index.html

```diff
@@ -4,9 +4,106 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="stylesheet" href="styles/snow.css">
+    <link rel="stylesheet" href="styles/rain_sun.css">
+    <link rel="stylesheet" href="styles/clouds.css">
+    <link rel="stylesheet" href="styles/select.css">
+    <link rel="stylesheet" href="styles/button.css">
+
+
+
+    <!-- bootstrap for buttons -->
+
+
     <title>Weather Report</title>
 </head>
-<body>
+<body class="default">
```

Comment:
Since there is only a single body tag putting the class `default` on it is redundant.